docs/developer_docs.md

```diff
@@ -22,7 +22,7 @@ This plugin integrates the services of Azure DevOps in Mattermost. For a stable
 This plugin contains the following features:
 - Right-hand sidebar (RHS) shows the list of linked projects and subscriptions created for each project with separate buttons to connect an account, link or unlink a project and add or delete a subscription.
 
-- Preview work item: A preview of the work item will be created when a work item URL is posted in a channel and the user account is connected.
+- Preview work item: A preview of the work item will be created when a work item URL is posted in a channel and the user account is connected to his Azure DevOps account.
```

Comment:
```suggestion
- Preview work item: A preview of the work item will be created when a work item URL is posted in a channel and the user is connected to his Azure DevOps account.
```
I think this will be better.